index.html

```diff
@@ -17,7 +17,7 @@
     <div id="navbar"></div>
     <header>
       <div class="main-name">
-        <img class="logo" src="logo.png" alt="" />
+        <img class="logo" src="slogo.png" alt="">

```

Comment:
Can you please keep the logo with the white background? Keep it as it is, that is, logo.png @9Shalini